sigridci/sigridci.py

```diff
@@ -140,7 +140,7 @@ def retry(self, operation, *, attempts=5, allow404=False, allowEmpty=True):
                     return response
             except urllib.error.HTTPError as e:
                 if e.code in [401, 403]:
```

Comment:
Nearly always when Sigrid returns a 401, the problem is authentication. In that case, Sigrid doesn't know who you are and hence cannot check whether you're authorized _for this system_. 
```suggestion
                if e.code == 401:
                    log(f"You are not authenticated to Sigrid (HTTP status {e.code}). Did you set environment variable SIGRID_CI_TOKEN to the correct token?")
                    sys.exit(1)
                elif e.code == 403:
```